README.md

```diff
@@ -58,83 +58,83 @@ Parameter|Value|Default|Description
 
 ### Outputs
 
-Output | Type | Description
----|---|---
-`germlineSVs`|File|Filtered vcf file containing structural variant calls
-`germlineCNVs`|File|Filtered vcf file containing copy number variant calls
+Output | Type | Description | Labels
+---|---|---|---
+`germlineSVs`|File|Filtered vcf file containing PASS structural variant calls|vidarr_label: germlineSVs
```

Comment:
This should also be reverted to not include "PASS"

Reply:
Also fixed